contract/src/main/java/com/ritense/valtimo/contract/form/FormFieldDataResolver.java

```diff
@@ -30,6 +30,15 @@ default boolean supports(String externalFormFieldType) {
         return supports(ExternalFormFieldType.fromKey(externalFormFieldType));
     }
 
-    Map<String, Object> get(String documentDefinitionName, UUID documentId, String... varNames);
+    @Deprecated(forRemoval = true, since = "9.21")
+    default Map<String, Object> get(String documentDefinitionName, UUID documentId, String... varNames) {
```

Comment:
What I meant was for the new method to be default, not this one. So:

```
    default Map<String, Object> get(
        String documentDefinitionName,
        UUID documentId,
        FormFieldDataResolverProperties additionalProperties,
        String... varNames
    ) {
        return get(documentDefinitionName, documentId, varNames);
    }
```

Reply:
doesn't that make the code breaking when you change the default

I think you should change the default when the old method is removed in the future

Reply:
The default would be to ensure backwards compatibility, the intent is still to override it in your implementation.